style.css

```diff
@@ -342,3 +349,101 @@ nav .status-bar {
   font-size: 22px;
   line-height: 43px;
 }
+
+.contact-section {
+  background: white;
+}
+
+.contact h3 {
+  color: #172b4d;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  padding-top: 120px;
+}
+
+.contact-list {
+  border-bottom: 2px solid #dbd8d7;
+  padding-bottom: 20px;
+}
```

Comment:
- [ ] Great job so far. However, there is a slight difference between your design and the Figma's( font, color, spacing).
- [ ] [_OPTIONAL_] It would be better to remove the li tags in the tag as it might be more semantic and also it might be the cause for the style difference.

Your Implementation | Figma Design
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/154056642-e1e1d020-c5f6-4f53-8112-b62253f3973f.png) | ![image](https://user-images.githubusercontent.com/59416136/154056733-6a9ac74a-3eba-40e4-a28d-f86a2bd2c00a.png)

Kindly fix this.